open_city_profile/admin_site.py

```diff
@@ -0,0 +1,11 @@
+from django.conf import settings
+from helusers.admin_site import AdminSite as HelUsersAdminSite
+
+
+class AdminSite(HelUsersAdminSite):
+    index_template = "admin/admin_index.html"
+
+    def each_context(self, request):
+        ret = super().each_context(request)
+        ret["version"] = settings.VERSION or "unknown"
+        return ret
```

Comment:
This kind of variable assign is a bit of a javascriptism. Even in JS it's not advisable. Also the return variable name could be `context`.

```suggestion
    def each_context(self, request):
        context = super().each_context(request)
        context["version"] = (
            settings.VERSION if settings.VERSION is not None else "unknown"
        )
        return context
```

Reply:
Changed.